dlf/plugins/oai/class.tx_dlf_oai.php

```diff
@@ -967,13 +967,9 @@ protected function verbIdentify() {
 
 		} else {
 
-			if (TYPO3_DLOG) {
-
-				t3lib_div::devLog('[tx_dlf_oai->verbIdentify()] Incomplete plugin configuration', $this->extKey, SYSLOG_SEVERITY_ERROR);
-
-			}
+			t3lib_div::devLog('[tx_dlf_oai->verbIdentify()] Incomplete plugin configuration', $this->extKey, SYSLOG_SEVERITY_ERROR);
```

Comment:
I agree with your proposal to add a helper function as wrapper for error handling, but that should IMHO be done in a separate pull request and for the whole extension at once.